metau-capstone/StudyViewController.m

```diff
@@ -205,9 +224,9 @@ - (void) loadFlashcard {
                 // Update lastFinished date
                 userObject[@"prevFinishedDate"] = dateString;
                 [userObject saveInBackground];
-                
-                // Increment day counter for the user
-                [userObject incrementKey:@"userDay"];
+             
+                userObject[@"phaseNum"] = @(3);
```

Comment:
since the number is difficult to understand, in this case the preferred way is to construct enums using `NS_ENUM` with type of NSInteger